spec/factories/user.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    email { 'test@wp.pl' }
+    password { 'qwe1234' }
+  end
+  FactoryBot.build(:user, :client)
+  FactoryBot.build(:user, :admin)
```

Comment:
This seems to be redundant

Reply:
@jandudulski I didn't  cast `admin` or other default `string` into` integer.` Firstly I deleted default "role" value , so it was a `nil`, later I had a possibility to cast `string ` to the `integer`.